qiita_pet/templates/sample_validation.html

```diff
@@ -14,9 +14,13 @@
 </style>
 {% end %}
 {% block content %}
-<h1>Sample Validation</h1><br>
-{% if input %}
-<form action="{% raw qiita_config.portal_dir %}/admin/sample_validation/" method="post" id="sample_validation_form"> 
+{% if input or error is not None %}
+  {% if error is not None %}
+    <div class="alert alert-danger" role="alert">

```

Comment:
Is there a better way to phrase this? Perhaps making them entirely separate if - elif? It seems odd to check for both and then immediately check for the latter and if so bail early, and if not do something entirely different?

Reply:
Thought about it but that will require a full rewrite of the method AKA IMOO not worth it.